pageserver/src/bin/pageserver.rs

```diff
@@ -262,7 +263,8 @@ fn start_pageserver(
         version(),
         launch_ts.to_string()
     );
-    set_build_info_metric(GIT_VERSION);
+    info!("build_tag: {BUILD_TAG}");
```

Comment:
Please join this with the earlier `info!` message. Rationale: we often search for the above line via `git-env:`.

Though, looking now, there's an extra `.to_string()`, and really all these three should be parameters for a message which is currently not found "Starting up".

I am not able to suggest, but something like:

```patch
-    info!(
-        "version: {} launch_timestamp: {}",
-        version(),
-        launch_ts.to_string()
-    );
-    info!("build_tag: {BUILD_TAG}");
+    info!(version=%version(), launch_timestamp=%launch_ts, build_tag=%BUILD_TAG, "Starting up");
```

Reply:
Done https://github.com/neondatabase/neon/pull/5576/commits/9810845dd83ee2db84541c3cb37a577b413e2ae3

Reply:
@koivunej do you feel strongly about formatted vs structured metadata here?

Reply:
For pageserver we want to use structured formatting if possible. I don't know about the other binaries the logs of which I look less.

The https://github.com/neondatabase/neon/commit/9810845dd83ee2db84541c3cb37a577b413e2ae3 is looking good enough.